NEWS.md

```diff
@@ -1,5 +1,8 @@
 # ggplot2 2.2.1.9000
 
+* Documentation now uses markdown thanks to roxygen2md::roxygen2md() (@karawoo,
```

Comment:
Generally, I don't bother listing changes that don't have some user facing impact